tests/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -236,4 +236,23 @@ public function testOnlyOwnCommentIsParsed(): void
             ->getColumn('col1')
             ->getComment());
     }
+    
+    public function testListTableIndexesWithIndexExpression(): void
+    {
+        $sql = <<<SQL
+CREATE TABLE company (
+    com_id int(4),
+    com_name text(15)
+);
+CREATE INDEX com_id_index ON company (com_id);
+CREATE INDEX com_name_5_index ON company (substr(com_name, 0, 5))
```

Comment:
Given that the test creates the table via plain SQL, I assume that the DBAL doesn't support such indices.